src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java

```diff
@@ -0,0 +1,129 @@
+package org.broadinstitute.hellbender.utils.variant;
+
+import htsjdk.variant.variantcontext.Allele;
+
+/**
+ * This class contains any constants (primarily FORMAT/INFO keys) in VCF files used by the GATK.
+ * Note that VCF-standard constants are in VCFConstants, in htsjdk.  Keys in header lines should
+ * have matching entries in GATKVCFHeaderLines
+ */
+public final class GATKVCFConstants {
+
+    //INFO keys
+    public static final String ALLELE_BALANCE_HET_KEY =             "ABHet";
+    public static final String ALLELE_BALANCE_HOM_KEY =             "ABHom";
+    public static final String ORIGINAL_AC_KEY =                    "AC_Orig"; //SelectVariants
+    public static final String BEAGLE_AC_COMP_KEY =                 "ACH"; //BeagleOutputToVCF
```

Comment:
We just removed the Beagle converters in GATK3, so if you want to follow suit we don't need a bunch of these keys.